app/uk/gov/hmrc/customsservicestatus/controllers/BaseCustomsServiceStatusController.scala

```diff
@@ -27,13 +28,13 @@ abstract class BaseCustomsServiceStatusController(cc: ControllerComponents)(impl
 
   implicit val logger: Logger = Logger(this.getClass.getName)
 
-  def validateJson[T](f: T => Future[Result])(implicit request: Request[JsValue], r: Reads[T]): Future[Result] =
+  def validateJson[T](errorResponse: ErrorResponse)(f: T => Future[Result])(implicit request: Request[JsValue], r: Reads[T]): Future[Result] =
     request.body.validate[T] match {
       case JsSuccess(t, _) => f(t)
-      case error @ JsError(errors) =>
+      case error @ JsError(_) =>
         logger.warn(
           s"""|Failed to validate JSON from request body: ${error.toLogFormat}""".stripMargin
         )
-        Future(BadRequest(s"$errors"))
+        Future(BadRequest(errorResponse.message))
```

Comment:
any reason why we want to remove s"$errors" ? 
this looks ok for now, but in future if we add more request fields to the body and validation fails for any reason then it returns "invalid state", may be its better to keep this method generic as before ?